SafeExamBrowser.SystemComponents.Contracts/ISystemInfo.cs

```diff
@@ -47,5 +47,10 @@ public interface ISystemInfo
 		/// The MAC Addres of the network addapter
 		/// </summary>
 		string MacAddress { get; }
+
+		/// <summary>
+		/// Provides the DeviceID information of the user's Plug and Play devices 
+		/// </summary>
+		string[] DeviceId { get; }
```

Comment:
I think this should at least be changed to DeviceIds, since there are almost always multiple devices? Even better would be PlugAndPlayDeviceIds or alike.

Reply:
PlugAndPlayDeviceIds  is a good name, perfect